### ISSUE TITLE
Feature request: Option to add control+alt+n shortcut to start menu icon instead of desktop

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#11262]
### Is your feature request related to a problem? Please describe.

Request from some corporate users who want to install NVDA for users who may need it, and have the control+alt+n shortcut to start it - but either don't want to, or due to corporate policy cannot, have a desktop icon.

### Describe the solution you'd like

A shortcut key can be added to the start menu icon of a program, and this can be used to start the program in the same way as the shortcut key in the desktop icon.  Therefore, if users had the option (and ideally the ability to set via a command line argument for batch installs), this would enable them to not add a desktop icon, and their users to still press control+alt+n to launch NVDA.

**Comment by Brian1Gaff** [comment]
Sounds like some lazy admins about, since surely its a security change to 
allow such a shortcut anyway, wherever it is placed.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by DrSooom** [comment]
I guess the one and only main reason for this is that the admins don't want having the NVDA shortcut on all desktops by default because most of the users don't need NVDA. Therefore the never should open it. And if the shortcut isn't located on the desktop, they cannot start it accidently. If they really want to start NVDA, they have to do this via the start menu.

Hmm… What will happen when non-NVDA users accidently press CTRL+ALT+N? It's the same thing when an Android user press and hold Vol+ and Vol- together for more than two seconds to start TalkBack. The Google search entries for this phenomenon are really funny. 😉

Back to the feature request: I prefer a command line flag for this option as it is special.

**Comment by feerrenrut** [comment]
There are a few differences between these two shortcuts (one targets nvda_slave the other targets nvda.exe), it would be good to have in code comments about why this is.

Probably the main reason the hotkey is set on the desktop shortcut is ease of modification by an end user, but if the desktop shortcut is missing it makes sense to set it on the start menu shortcut.

Related issue **"Allow unicode in shortcut names created by installer. #11003 "**

**Comment by omatsei** [comment]
I was the one that requested this.

The purpose is for a multi-user system. We can't have NVDA start by default, and we don't want to put an icon on the desktop. However, after a user logs in, it would be very convenient to have them press Ctrl-Alt-N to start NVDA without having to navigate the Start menu (which can get VERY full). 

A command-line install option is fine, actually preferred for us.

**Comment by burakyuksek** [comment]
I would recommend that for personal use, I don't want the desktop icon and delete it, then ı make a shortcut to start menu icon manually.
